### DISCUSSION TITLE
tokyonight day scheme readability

### DISCUSSION THREAD

**Discussion by dvic** [folke/flash.nvim#130] [Q&A]
The jump highlights are show as follows with tokyonight day:

<img width="474" alt="image" src="https://github.com/folke/flash.nvim/assets/1214337/7dc963e0-97aa-4647-a598-c2600e7b6711">

Is this expected? The red background makes it hard to read. I consider myself a neovim beginner and I'm not sure where to look to change this ^^ Is it in flash.nvim or in the tokyonight scheme?

Thanks :)

**Comment by gsinclair** [comment]
Agreed; I find the hints unreadable in tokyonight dark mode (whatever the default one is) too. I came here looking to see if it had been mentioned.

**Comment by zoomido** [comment]
Yes I came here for this reason too. Impossible to see labels in this theme. If anyone has a workaround it would be great!

**Comment by modille** [comment]
I liked leap's label highlighting (a bold magenta fg), so I copied that:

```lua
{
  "folke/tokyonight.nvim",
  opts = {
    on_highlights = function(hl, c)
      hl["FlashLabel"] = { fg = c.magenta2, bold = true }
    end,
  },
}
```

day:

<img width="976" alt="image" src="https://github.com/user-attachments/assets/73055e35-3ec4-4bc7-820c-f802a330d965">

moon:

<img width="976" alt="image" src="https://github.com/user-attachments/assets/c99d2a28-cac9-4d13-ba21-4aeb35d5c581">

**Reply by zhousy29** [reply]
thanks, this really helps my eyes

**Comment by rbhanot4739** [comment]
I did open an [issue](https://github.com/folke/tokyonight.nvim/issues/683) in tokyonight repo about this few weeks back.
